src/cascadia/TerminalControl/ControlCore.cpp

```diff
@@ -594,9 +594,34 @@ namespace winrt::Microsoft::Terminal::Control::implementation
 
         _terminal->SetFontInfo(_actualFont);
 
-        // TODO: MSFT:20895307 If the font doesn't exist, this doesn't
-        //      actually fail. We need a way to gracefully fallback.
-        _renderer->TriggerFontChange(newDpi, _desiredFont, _actualFont);
+        if (_renderEngine)

```

Comment:
Maybe just return early when `_renderEngine` is NULL? That means it's being destroyed, right? I think

Reply:
I think it can also mean that `Initialize` has not been called yet!